### PR TITLE
Fix text-align in map marker

### PR DESCRIPTION
**Pull request by ildar170975**
## Proposed change

Before:
image

After:
image

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes # https://github.com/home-assistant/frontend/issues/20723
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Review by MindFreeze — CHANGES_REQUESTED** [review]
I don't think this selector is necessary. Can we just put `text-align: center` on `.marker`

**Comment by ildar170975** [comment]
> I don't think this selector is necessary. Can we just put `text-align: center` on `.marker`

Yes, will not not harm...

**Comment by MindFreeze** [comment]
> > I don't think this selector is necessary. Can we just put `text-align: center` on `.marker`
> 
> Yes, will not harm... I added a selector just for perfection since `text-align` is not needed for an image... If you insist - I will remove the selector.

Please do. Less selectors is better

**Comment by ildar170975** [comment]
> Please do. Less selectors is better

Done !!! ))

File: src/components/map/ha-entity-marker.ts [hunk 1]
```diff
@@ -56,6 +56,9 @@ class HaEntityMarker extends LitElement {
       .marker.picture {
         overflow: hidden;
       }
+      .marker {
```
Comment (MindFreeze) [inline src/components/map/ha-entity-marker.ts:59]: There is already`.marker` above on line 43. Just add it there

**Review by MindFreeze — DISMISSED** [review]

File: src/components/map/ha-entity-marker.ts (hunk 1 above)
Comment (ildar170975) [reply src/components/map/ha-entity-marker.ts]: Sorry, was fast and hence inaccurate..

**Review by piitaya — APPROVED** [review]

**Review by piitaya — APPROVED** [review]

**Pull request merged**